pkg/defaults/mesh/mesh_test.go

```diff
@@ -30,9 +30,43 @@ var _ = Describe("EnsureDefaultMeshResources", func() {
 		Expect(err).ToNot(HaveOccurred())
 	})
 
+	It("should create default secret resource", func() {
+		// when
+		err := mesh.EnsureDefaultMeshResources(context.Background(), resManager, defaultMesh, []string{}, context.Background(), false)
+		Expect(err).ToNot(HaveOccurred())
+
+		// and Dataplane Token Signing Key for the mesh exists
+		err = resManager.Get(context.Background(), system.NewSecretResource(), core_store.GetBy(tokens.SigningKeyResourceKey(issuer.DataplaneTokenSigningKeyPrefix(model.DefaultMesh), tokens.DefaultKeyID, model.DefaultMesh)))
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	It("should not create default policies", func() {
```

Comment:
kind of a nit but IMO the test case could be named better, like reading this `It` and the next `It` are just confusing together since they're opposite. I guess this one should be "should not create... if resources are already present"?